ale_linters/matlab/mlint.vim

```diff
@@ -0,0 +1,59 @@
+" Author: awlayton <alex@layton.in>
+" Description: mlint for MATLAB files
+
+if exists('g:loaded_ale_linters_matlab_mlint')
```

Comment:
We don't need the `exists` guards any more. They are being removed to aid some testing, and we should only be loading these files at most once per filetype anyway, I believe.